src/commands/createWebApp/setAppWizardContextDefault.ts

```diff
@@ -5,28 +5,30 @@
 
 import * as fse from 'fs-extra';
 import * as path from 'path';
-import { ConfigurationTarget, workspace, WorkspaceConfiguration, WorkspaceFolder } from 'vscode';
+import { ConfigurationTarget, workspace, WorkspaceFolder } from 'vscode';
 import { IAppServiceWizardContext, LinuxRuntimes, WebsiteOS } from 'vscode-azureappservice';
 import { IActionContext, LocationListStep } from 'vscode-azureextensionui';
-import { configurationSettings, extensionPrefix } from '../../constants';
+import { configurationSettings } from '../../constants';
 import { javaUtils } from '../../utils/javaUtils';
-import { findFilesByFileExtension, getContainingWorkspace } from '../../utils/workspace';
+import { findFilesByFileExtension } from '../../utils/workspace';
+import { getWorkspaceSetting } from '../../vsCodeConfig/settings';
 
 export interface IDeployWizardContext extends IActionContext {
-    fsPath?: string;
+    workspace: WorkspaceFolder;
     deployedWithConfigs?: boolean;
     configurationTarget?: ConfigurationTarget;
 }
 
-export async function setAppWizardContextDefault(wizardContext: IAppServiceWizardContext & IDeployWizardContext): Promise<void> {
+export async function setAppWizardContextDefault(wizardContext: IAppServiceWizardContext, deployedWorkspace?: WorkspaceFolder): Promise<void> {
```

Comment:
`wizardContext: IAppServiceWizardContext & Partial<IDeployWizardContext>`

and then you should be able to use `wizardContext.workspace`